P5/Source/Specs/eventName.xml

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="UTF-8"?><!--
```

Comment:
Please insert a newline so that the comment starts on line 2.
(Not important, but we have been trying to keep the XML declarations, initial comments, initial PIs, and first start tag all on lines by themselves. Of the 866 files in Source/ that have an XML declaration, it is on a line by itself in all but 20.)